src/simulation.jl

```diff
@@ -14,28 +14,88 @@ end
 
 """
     simulate(
-    [rng::AbstractRNG,]
+    rng::AbstractRNG,
     design::AbstractDesign,
     signal,
     onset::AbstractOnset,
     noise::AbstractNoise = NoNoise();
-    return_epoched=false,
+    return_epoched = false,
     )
 
-Main simulation function, given `Design`, [Array of] `Component`, `Onset` and optional [`Noise`], returns continuous or epoched signal.
-
-## optional
-- `return_epoched` (Bool, default: `false`):  Skip the Onset-calculation and conversion to continuous data and return the epoched data directly (see also remarks below).
-
-## Return 
-Depending on the design, the components and on `return_epoched`, the output can be a 1-D, 2-D, 3-D or 4-D Array. For example, a 4-D Array would have the dimensions `channels x time x trials x subjects`
+	simulate(
+    design::AbstractDesign,
+    signal,
+    onset::AbstractOnset,
+    noise::AbstractNoise = NoNoise();
+    return_epoched = false,
+    )
 
-## Notes
+Return continuous or epoched signal, given `Design`, [Array of] `Component`, `Onset` and 
+optional [`Noise`] and [`RNG`]. Main simulation function.
+
+# Arguments
+- `design::AbstractDesign`: Desired experimental design.
+- `signal`: `Component` for the desired signal.
+- `onset::AbstractOnset`: Desired onset.
+- `noise::AbstractNoise = NoNoise()` (optional): Desired noise.
+- `rng::AbstractRNG` (optional): Random number generator, important to ensure reproducibility.
+
+# Keyword arguments
+- `return_epoched = false`: Skip the Onset-calculation and conversion to continuous data 
+    and return the epoched data directly (see also Notes below).
+
+# Returns
+- `signal` : Generated signal. Depending on the design, on the components and on 
+    `return_epoched`, the output can be a 1-D, 2-D, 3-D or 4-D Array. 
+    For example, a 4-D Array would have the dimensions `channels x time x trials x subjects`.
+- `events`: Generated events.
+
+# Examples
+Adapted from the quickstart tutorial in the UnfoldSim docs.
+```julia-repl
+julia> using UnfoldSim
```

Comment:
For the examples, you can omit the `using UnfoldSim` since the examples are for UnfoldSim functions.